src/HeaderRow.js

```diff
@@ -26,9 +34,13 @@ var HeaderRow = React.createClass({
     height: PropTypes.number.isRequired,
     columns: PropTypes.arrayOf(React.PropTypes.shape(ExcelColumn)).isRequired,
     onColumnResize: PropTypes.func,
+    onSort: PropTypes.func.isRequired,
     style: PropTypes.shape(HeaderRowStyle)
+
   },
 
+  mixins: [ColumnUtilsMixin],
```

Comment:
is there much point this being a mixin, rather than just a lib? ie `this.getColumn(this.props.columns, i);` as a mixin vs `ColumnUtils.getColumn(this.props.columns, i);` as a lib? means we are 1 step cloiser to es6 classes